src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -0,0 +1,107 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.math.controller.ArmFeedforward;
```

Comment:
Minor, but does anyone know of an FRC-specific Java styleguide? Here's Google's on how to sort imports (https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing), where I think this section should be alphabetized. (Often you can get VSCode to handle this for you given the right settings.)